.mci.yml

```diff
@@ -960,6 +960,20 @@ tasks:
         - func: "run_kms_servers"
         - func: "test"
 
+    - name: calc_release_version_selftest
```

Comment:
The additional step in the release process and additional task in CI may add some maintainence burden.

IMO the self test is helpful to verify changes in `calc_release_version.py`. I expect the self tests may not be helpful for the vast majority of all other commits.

I suggest:
- Remove the task from Evergreen and removing the updates to the release process.
- Add a comment in `calc_release_version.py` to suggest running the self test when making an update. Suggestion:
```python
# Updates to calc_release_version.py can be manually tested by running the calc_release_version_selftest.sh script
```

Reply:
Good point about the maintenance burden. That thought did cross my mind, but then I wasn't entirely convinced one way or the other. I can definitely agree with removing it from CI and documenting a manual test procedure inside of `calc_release_version.py`. I'll work on getting that updated soon.

Reply:
Done